### PR TITLE
New header for ECR viewer

### LINKED ISSUE #2064: Add a header to the eCR Viewer
## Action Requested
Like the title says!

## Acceptance Criteria

- [ ]  The eCR viewer page has a header matching designs
- [ ]  The header DOES NOT include the little DIBBs logo to the left of the DIBBS text. We don't have permission to use it yet.

## Testing
Just snapshots

## Designs
https://www.figma.com/design/tXWUeV3Kjehd7VNqHojZYq/eCR-Viewer?node-id=3160-36760&t=GKqK6VqHnnkFkSSL-0

### PR DESCRIPTION
**Pull request by nickbristow**
# PULL REQUEST

## Summary
Adds header.
Note, no header image because of things
## Related Issue
Fixes #2064

## Additional Information
Anything else the review team should know?

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

**Comment by sarahtress** [comment]
@nickbristow can you adjust the padding so that "eCR Viewer" is in line with the left-most elements in the content below? tried to draw on the screenshot to show what I mean in the attached image

also, while we wait for logo approval, can the text read "DIBBs eCR Viewer"?

346227558-812cfcbd-0174-4892-848c-8b20941e2b5b-1

**Comment by sarahtress** [comment]
@nickbristow spacing looks good! can you change the text to be "DIBBs eCR Viewer" too? thanks!

**Review by JNygaard-Skylight — CHANGES_REQUESTED** [review]

File: containers/ecr-viewer/src/app/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * @returns The header section of the application.
+ */
+const Header: React.FC = () => (
+  <header className="usa-header usa-header--basic">
+    <div className="usa-nav-container padding-left-0">
+      <div className="usa-navbar">
+        <div className="usa-logo">
+          <em className="usa-logo__text">
+            <a href="/" title="<Project title>">
+              eCR Viewer
```
[the remark this answers is not in the document]
Comment (JNygaard-Skylight) [reply containers/ecr-viewer/src/app/Header.tsx:17]: Also, making "eCR Summary" a `<h2>` will include it in the side nav, fixing #2071.

[the remark this answers is not in the document]
Comment (nickbristow) [reply containers/ecr-viewer/src/app/Header.tsx]: good call

[the remark this answers is not in the document]
Comment (nickbristow) [reply containers/ecr-viewer/src/app/Header.tsx]: resolved

[the remark this answers is not in the document]
Comment (JNygaard-Skylight) [reply containers/ecr-viewer/src/app/Header.tsx]: This did not resolve #2071, but the header hierarchy is fixed, so this PR is good.

**Pull request merged**